packages/cli/package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "@synthql/cli",
+    "version": "0.0.1",
+    "repository": {
+        "type": "git",
+        "url": "git@github.com:synthql/synthql.git"
+    },
+    "license": "MIT",
+    "scripts": {
+        "build": "yarn tsc",
```

Comment:
Please be consistent with other packages unless there is a good reason to deviate: We use `tsc` just for type checking and `vite` for building/bundling.

If you look at the package.json of other packages you will notice

Reply:
Indeed. I did do this at the beginning, but I was running into some errors due to Vite struggling with bundle sharing/compatibility issues, showing some errors about how some of the modules weren't available outside of the browser. But I will set it up again now, let's loot at fixing the errors.